tests/js/setup-before-after.js

```diff
@@ -32,6 +32,10 @@ import { enabledFeatures } from '../../assets/js/features';
 // It is re-set here since fetch-mock-jest must be imported during Jest's `setupFilesAfterEnv` or later.
 global.fetchMock = fetchMockJest;
 
+beforeAll( () => {
+	jest.useRealTimers();
+} );
```

Comment:
I've added a comment about this. It would be more work and more problematic to rely on fake timers everywhere as we'd need to manually advance them for every resolver call.

https://github.com/google/site-kit-wp/blob/07e66cac79329d893ccbad2d11e2ecf21ecf9249/tests/js/setup-before-after.js#L38-L43

As for running pending timers, we already have that covered in a global `afterEach` -

https://github.com/google/site-kit-wp/blob/07e66cac79329d893ccbad2d11e2ecf21ecf9249/tests/js/setup-before-after.js#L64-L74